src/validations/taskValidator.js

```diff
@@ -0,0 +1,174 @@
+const Joi = require('joi');
+const MyValidator = require('./validator');
+
+
+const schemas = {
+    createTask: {
+        body: Joi.object().keys(
+            {
+                taskName: Joi.string()
+                    .regex(/^[a-zA-Z0-9_ ]{3,100}$/)
+                    .messages({
+                        "string.pattern.base": "Task name not valid"
+                    })
+                    .required(),
+                assignee: Joi.string()
+                    .hex()
+                    .length(24)
+                    .allow(null, '')
+                    .messages({
+                        "string.pattern.base": "User not valid"
+                    }),
+                content: Joi.string()
+                    .min(3)
+                    .max(999)
+                    .required()
+                    .messages({
+                        "string.min": "must more than 2 characters.",
+                        "string.max": "must less than 1000 characters."
+                    }),
+                attachments: Joi.any(),
+                point: Joi.number()
+                    .min(1)
+                    .max(10)
+                    .required()
+                    .messages({
+                        "string.min": "must more than 0 characters.",
+                        "string.max": "must less than or equal 10 characters.",
+                        "string.pattern.base": "Format must in todo|doing|done|rejected"
+                    }),
+                deadlineAt: Joi.date()
+                    .greater(Date.now())
```

Comment:
chỗ này e phải trừ đi 1 khoảng time nữa, ví dụ: ở client nó chọn hiện tại là 1699251615827 , FE gọi API mất 3s thì khi BE nhận được là 1699251618827, sẽ ko đúng lắm. E double check thử xem